CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+[Docusaurus](https://docusaurus.io/) is a static site generator that helps you ship beautiful, accessible docs. For building our documentation engine, we have made certain modifications to the template generated by [Docusaurus](https://docusaurus.io) to be able to properly showcase all our contents with their respective versions.
+
+### Built With
+
+- [Docusaurus](https://docusaurus.io/)
+- [React](https://reactjs.org/)
+- [Tailwind](https://tailwindcss.com/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This section describes how you can get our documentation portal up and running on your machine.
+
+### Prerequisites
+
+- [node](https://nodejs.org/en/)
+- [npm](https://www.npmjs.com/)
+
+### Installation
+
+1. Clone the repo
+
+```shell
+git clone https://github.com/keptn-sandbox/new-keptn-docs-engine
+```
+
+2. Install NPM packages
+
+```shell
+npm install
+```
+
+3. Run the app
+
+```shell
+npm start
```

Comment:
So this is for building changes I make to the doc locally before I push them?  How do I then view the changes locally?

Reply:
`npm start` does both i.e. build and preview the doc site locally. If we want only build then we have to run `npm run build`.

Reply:
what is the min node version to build this successfully, or do we recommend a node version for building the page? 

would it be worth to also provide `make` targets, with a docker image, to remove the dependency of finding the right node version and setting it up properly?

Reply:
@aepfli to build we need Node.js version 16.14 or above. Yeah, we can dockerize this but I feel it will be cumbersome to maintain instead we can directly mention the requirement in README.